docker-compose.yml

```diff
@@ -50,5 +46,20 @@ services:
     deploy:
       replicas: ${DELAYED_JOB_REPLICAS:-1}
 
+  traefik:
+    image: traefik:v3.2
+    command:
+      - --configFile=/traefik.yml
+    ports:
+      - ${WEB_BIND:-8080}:8080
+      - 127.0.0.1:8280:8080 # Dashborad
```

Comment:
```suggestion
      - 127.0.0.1:8280:8080 # Dashboard
```

Reply:
Moreover is it necessary to expose port 8280 if dashboard is disabled in traefik.yml ?

Reply:
I do this way because it avoids extra config when dashboard is enabled